
Does going to university in a different country affect your mental health? A Japanese international university surveyed its students in 2018 and published a study the following year that was approved by several ethical and regulatory boards.

The study found that international students have a higher risk of mental health difficulties than the general population, and that social connectedness (belonging to a social group) and acculturative stress (stress associated with joining a new culture) are predictive of depression.


Exploring the `students` dataset using PostgreSQL to determine whether I would come to a similar conclusion for international students and see if the length of stay is a contributing factor.

Here is a data description of the columns that may be helpful for the analysis.

| Field Name    | Description                                      |
| ------------- | ------------------------------------------------ |
| `inter_dom`     | Types of students (international or domestic)   |
| `japanese_cate` | Japanese language proficiency                    |
| `english_cate`  | English language proficiency                     |
| `academic`      | Current academic level (undergraduate or graduate) |
| `age`           | Current age of student                           |
| `stay`          | Current length of stay in years                  |
| `todep`         | Total score of depression (PHQ-9 test)           |
| `tosc`          | Total score of social connectedness (SCS test)   |
| `toas`          | Total score of acculturative stress (ASISS test) |

In [1]:
-- Run this code to save the CSV file as students
SELECT * 
FROM 'students.csv';

inter_dom region  gender  ... alone_bi  others_bi  internet_bi
0       Inter    SEA    Male  ...       No         No           No
1       Inter    SEA    Male  ...       No         No           No
2       Inter    SEA    Male  ...       No         No           No
3       Inter     EA  Female  ...       No         No           No
4       Inter     EA  Female  ...       No         No           No
..        ...    ...     ...  ...      ...        ...          ...
281       NaN    NaN     NaN  ...      NaN        NaN          NaN
282       NaN    NaN     NaN  ...      NaN        NaN          NaN
283       NaN    NaN     NaN  ...      NaN        NaN          NaN
284       NaN    NaN     NaN  ...      NaN        NaN          NaN
285       NaN    NaN     NaN  ...      NaN        NaN          NaN

[286 rows x 50 columns]

The following are the exploratory steps.

I start by counting all of the records in the dataset.

In [2]:
SELECT COUNT(*)
FROM 'students.csv';

count_star()
0           286

Then, I count all records perstudent type to see how the records are categorized and scored.

In [3]:
SELECT inter_dom, COUNT(inter_dom) AS count_inter_dom, COUNT(*) AS total_records
FROM 'students.csv'
GROUP BY inter_dom;

inter_dom  count_inter_dom  total_records
0     Inter              201            201
1       Dom               67             67
2       NaN                0             18

There are 201 records are international and 67 are domestic. However, the table also has 18 with NULL value.

Next, I will filter the data to see how it differs between the students types.

In [4]:
SELECT *
FROM 'students.csv'
WHERE inter_dom = 'Inter';

inter_dom region  gender  ... alone_bi  others_bi  internet_bi
0       Inter    SEA    Male  ...       No         No           No
1       Inter    SEA    Male  ...       No         No           No
2       Inter    SEA    Male  ...       No         No           No
3       Inter     EA  Female  ...       No         No           No
4       Inter     EA  Female  ...       No         No           No
..        ...    ...     ...  ...      ...        ...          ...
196     Inter    SEA    Male  ...      Yes         No           No
197     Inter    SEA  Female  ...      Yes         No           No
198     Inter    SEA  Female  ...       No         No           No
199     Inter    SEA  Female  ...      Yes        Yes          Yes
200     Inter     EA  Female  ...       No         No           No

[201 rows x 50 columns]

In [5]:
SELECT *
FROM 'students.csv'
WHERE inter_dom = 'Dom';

inter_dom region  gender academic  ...  religion_bi  alone_bi  others_bi internet_bi
0        Dom    JAP  Female     Grad  ...          Yes        No         No          No
1        Dom    JAP  Female    Under  ...           No        No         No          No
2        Dom    JAP  Female    Under  ...           No        No         No          No
3        Dom    JAP    Male    Under  ...           No        No         No          No
4        Dom    JAP  Female    Under  ...           No        No         No          No
..       ...    ...     ...      ...  ...          ...       ...        ...         ...
62       Dom    JAP  Female    Under  ...           No        No         No         Yes
63       Dom    JAP  Female    Under  ...           No        No         No          No
64       Dom    JAP  Female    Under  ...           No        No         No          No
65       Dom    JAP    Male    Under  ...           No        No         No          No
66       Dom    JAP    Male    Under  ...           No       Yes         No          No

[67 rows x 50 columns]

In [6]:
SELECT *
FROM 'students.csv'
WHERE inter_dom IS NULL;

inter_dom region gender academic  ...  religion_bi  alone_bi  others_bi internet_bi
0        NaN    NaN    NaN      NaN  ...          NaN       NaN        NaN         NaN
1        NaN    NaN    NaN      NaN  ...           19        65         21          45
2        NaN    NaN    NaN      NaN  ...          249       203        247         223
3        NaN    NaN    NaN      NaN  ...          NaN       NaN        NaN         NaN
4        NaN    NaN    NaN      NaN  ...           19        65         21          45
5        NaN    NaN    NaN      NaN  ...          249       203        247         223
6        NaN    NaN    NaN      NaN  ...          NaN       NaN        NaN         NaN
7        NaN    NaN    NaN      NaN  ...          NaN       NaN        NaN         NaN
8        NaN    NaN    NaN      NaN  ...          NaN       NaN        NaN         NaN
9        NaN    NaN    NaN      NaN  ...          NaN       NaN        NaN         NaN
10       NaN    NaN    NaN      NaN  ...          NaN       NaN        NaN         NaN
11       NaN    NaN    NaN      NaN  ...          NaN       NaN        NaN         NaN
12       NaN    NaN    NaN      NaN  ...          NaN       NaN        NaN         NaN
13       NaN    NaN    NaN      NaN  ...          NaN       NaN        NaN         NaN
14       NaN    NaN    NaN      NaN  ...          NaN       NaN        NaN         NaN
15       NaN    NaN    NaN      NaN  ...          NaN       NaN        NaN         NaN
16       NaN    NaN    NaN      NaN  ...          NaN       NaN        NaN         NaN
17       NaN    NaN    NaN      NaN  ...          NaN       NaN        NaN         NaN

[18 rows x 50 columns]

As we can see with international student type, we have students are in both undergraduate and graduate academic levels with age range is from 17 to over 30. On the other hand, students who do not study aboard are all undergraduate students, and all of them are under 30.

With NULL records, there are clearly no valuable information.

I am going to find the summary statistics of the diagnostic tests for all students.

In [2]:
SELECT 'Average', ROUND(AVG(todep),2) AS 'Score of depression (PHQ-9 test)', ROUND(AVG(tosc), 2) AS 'Score of social connectedness (SCS test)', ROUND(AVG(toas), 2) AS 'Score of acculturative stress (ASISS test)'
FROM 'students.csv'
UNION
SELECT 'Standard deviation', ROUND(STDDEV(todep), 2) AS 'Score of depression (PHQ-9 test)', ROUND(STDDEV(tosc), 2) AS 'Score of social connectedness (SCS test)', ROUND(STDDEV(toas),2) AS 'Score of acculturative stress (ASISS test)'
FROM 'students.csv'
UNION
SELECT 'Max', MAX(todep) AS 'Score of depression (PHQ-9 test)', MAX(tosc) AS 'Score of social connectedness (SCS test)', MAX(toas) AS 'Score of acculturative stress (ASISS test)'
FROM 'students.csv'
UNION
SELECT 'Min', MIN(todep) AS 'Score of depression (PHQ-9 test)', MIN(tosc) AS 'Score of social connectedness (SCS test)', MIN(toas) AS 'Score of acculturative stress (ASISS test)'
FROM 'students.csv';

'Average'  ...  Score of acculturative stress (ASISS test)
0                 Max  ...                                      145.00
1  Standard deviation  ...                                       22.64
2             Average  ...                                       72.38
3                 Min  ...                                       36.00

[4 rows x 4 columns]

I am going to repeat the summary statistics steps but this time only applies to international students.

In [1]:
SELECT 'Average', ROUND(AVG(todep),2) AS 'Score of depression (PHQ-9 test)', ROUND(AVG(tosc), 2) AS 'Score of social connectedness (SCS test)', ROUND(AVG(toas), 2) AS 'Score of acculturative stress (ASISS test)'
FROM 'students.csv'
WHERE inter_dom = 'Inter'
UNION
SELECT 'Standard deviation', ROUND(STDDEV(todep), 2) AS 'Score of depression (PHQ-9 test)', ROUND(STDDEV(tosc), 2) AS 'Score of social connectedness (SCS test)', ROUND(STDDEV(toas),2) AS 'Score of acculturative stress (ASISS test)'
FROM 'students.csv'
WHERE inter_dom = 'Inter'
UNION
SELECT 'Max', MAX(todep) AS 'Score of depression (PHQ-9 test)', MAX(tosc) AS 'Score of social connectedness (SCS test)', MAX(toas) AS 'Score of acculturative stress (ASISS test)'
FROM 'students.csv'
WHERE inter_dom = 'Inter'
UNION
SELECT 'Min', MIN(todep) AS 'Score of depression (PHQ-9 test)', MIN(tosc) AS 'Score of social connectedness (SCS test)', MIN(toas) AS 'Score of acculturative stress (ASISS test)'
FROM 'students.csv'
WHERE inter_dom = 'Inter';

'Average'  ...  Score of acculturative stress (ASISS test)
0                 Max  ...                                      145.00
1             Average  ...                                       75.56
2  Standard deviation  ...                                       22.56
3                 Min  ...                                       36.00

[4 rows x 4 columns]

International students seem to have the same diagnostic scores as total records, with the average score of depression on the PHQ-9 test at 8.04 compared to 8.19 for all records. The average score of social connectedness on the SCS test is 37.42 compared to 37.47, and the average score of acculturative stress in ASISS test is 75.56 compared to  72.38. However, those information tests do not give accurate information for us to come up with a conclusion because the number of international students is the vast majority in the dataset, 75%.

To see whether the length of stay impacts the average diagnostic scores for international students, this is the final step of the analysis to answer the question if international students have a higher risk of mental health difficulties than the general population.

In [9]:
SELECT stay, ROUND(AVG(todep),2) AS average_phq, ROUND(AVG(tosc), 2) AS average_scs, ROUND(AVG(toas), 2) AS average_as
FROM 'students.csv'
WHERE inter_dom = 'Inter'
GROUP BY stay
ORDER BY stay DESC;

stay  average_phq  average_scs  average_as
0    10        13.00        32.00       50.00
1     8        10.00        44.00       65.00
2     7         4.00        48.00       45.00
3     6         6.00        38.00       58.67
4     5         0.00        34.00       91.00
5     4         8.57        33.93       87.71
6     3         9.09        37.13       78.00
7     2         8.28        37.08       77.67
8     1         7.48        38.11       72.80

In conclusion, the longer international students stay away from home, the higher chance they will struggle with mental health problems. The average score of depression increased from 7.48 in the first year to 13 in the 10th year, which increased by 73.8%. 

To answer whether going to university in a different country affects your mental health. The answer is YES. International students who live far away from their home countries tend to face a higher risk of mental health difficulties than the general population. 

Here are some resources that international students may find helpful in order to help them overcome their mental health struggles:

1. "How to look after your mental health as an international student" https://studee.com/guides/how-to-look-after-your-mental-health-as-an-international-student/
2. "6 Mental Health Tips for International Students" https://www.usnews.com/education/best-colleges/articles/2019-04-08/how-to-look-after-mental-health-as-an-international-studentin-the-us
3. "Mental Health Advice for International Students" https://shorelight.com/student-stories/mental-health-advice-for-international-students/

